baseplate/clients/requests.py

```diff
@@ -101,6 +107,35 @@ def http_adapter_from_config(
     return ValidatingHTTPAdapter(**kwargs)
 
 
+PROM_NAMESPACE = "http_client"
+HTTP_LABELS_COMMON = [
+    "http_method",
+    "http_client_name",
+]
+HTTP_LABELS_TERMINAL = [*HTTP_LABELS_COMMON, "http_success"]
+
+# Latency histogram of HTTP calls made by clients
+# buckets are defined above (from 100µs to ~14.9s)
+LATENCY_SECONDS = Histogram(
+    f"{PROM_NAMESPACE}_latency_seconds",
+    "Latency histogram of HTTP calls made by clients",
+    HTTP_LABELS_TERMINAL,
```

Comment:
Why aren't we tracking response_code here?

Reply:
The spec says histograms don't get `http_response_code`. Cardinality would be the reason. I think I already said this in a comment above but these histograms are 15 or 16 series each (because of the buckets). Times all the http response codes it explodes really quickly.